README.md

```diff
@@ -1,2 +1,55 @@
-# qmdataset
-A collection of QM data for training potential functions
+# SPICE: A Dataset for Training Machine Learning Potentials
+
+SPICE (Small-Molecule/Protein Interaction Chemical Energies) is a collection of quantum mechanical data for
+training potential functions.  The emphasis is particularly on simulating drug-like small molecules interacting
+with proteins.  It is designed to achieve the following goals.
+
+- **Cover a wide range of chemical space**.  It includes 17 elements and a wide range of chemical groups.
+  It includes charged and polar molecules as well as neutral ones.  It is designed to sample a wide range of
```

Comment:
Maybe we should emphasize the charged molecules more. It maybe be the main advantage of SPICE.

Reply:
What else do you suggest saying to emphasize them more?